### ISSUE TITLE
One Click App: Big Blue Button

### ISSUE THREAD

**Issue by bbodensieck** [caprover/caprover#943]
During COVID-19 pandemic many teachers are teaching remotely from home. The tools which are supported by schools sometimes are not state of the art. Self-hosting a platform is only possible for teachers with technical background.

- It would by awesome to host Big Blue Button as a learning/teaching platform for students. Via One Click App it would be easy to set up for teachers without any technical background.

- As an alternative it would be awesome to get the One Click App "Jitsi" working via HTTPS without any problems. I have tried it several times but was not able to get it work publicly. Is there probably a docker-tag which was tested in CapRover by the community?

To sum up: It would be great to provide a One Click App for learning/teaching/conferences which really works out of the box without fighting against issues.

**Comment by githubsaturn** [comment]
Duplicate of this: https://github.com/caprover/one-click-apps/issues/307

Please note that one click apps are community created/maintained. Please feel free to ping anyone in "Big Blue Button" community to create a one click app for it. The template is very similar to Docker compose template and the instructions are here: https://github.com/caprover/one-click-apps#how-to-create-a-one-click-app-as-of-v180
